src/components/ftbot/TradeListNav.vue

```diff
@@ -9,26 +9,49 @@
         >Trade Navigation {{ sortNewestFirst ? '&#8595;' : '&#8593;' }}
       </b-list-group-item>
       <b-list-group-item
-        v-for="trade in sortedTrades"
+        v-for="(trade, i) in sortedTrades"
         :key="trade.open_timestamp"
         button
-        class="d-flex flex-wrap justify-content-between align-items-center py-1"
+        class="d-flex flex-column py-1"
         :title="`${trade.pair}`"
         :active="trade.open_timestamp === selectedTrade.open_timestamp"
         @click="onTradeSelect(trade)"
       >
-        <div>
-          <span v-if="botStore.activeBot.botState.trading_mode !== 'spot'">{{
-            trade.is_short ? 'S-' : 'L-'
-          }}</span>
-          <DateTimeTZ :date="trade.open_timestamp" />
+        <div class="d-flex">
+          <div class="d-flex flex-column">
+            <div>
+              <span v-if="botStore.activeBot.botState.trading_mode !== 'spot'">{{
+                trade.is_short ? 'S-' : 'L-'
+              }}</span>
+              <DateTimeTZ :date="trade.open_timestamp" />
+            </div>
+            <TradeProfit :trade="trade" class="my-1" />
+            <ProfitPill
+              v-if="backtestMode"
+              :profit-ratio="trade.profit_ratio"
+              :stake-currency="botStore.activeBot.stakeCurrency"
+            />
+          </div>
+          <div class="d-flex flex-fill justify-content-end">
```

Comment:
I think you can omit this div - it's only content is the button - and layout doesn't really change with/without it.

i think the button should be pushed all the way to the right though - it's kinda weird to have the button somewhere int he middle (see below). 

![image](https://github.com/freqtrade/frequi/assets/5024695/3aad4ca5-e320-4094-a631-ee24467c8b3e)

maybe also a very light border around the orders - so there's a clear offset to show that "this is something else" ... ?  (it may also look bad though - i didn't really experiment with how this might look).

Reply:
Interesting, so it wouldn't look like this is why I used the div, and previously there was an issue with the timestamp, profit and orders basically becoming 3 columns when wider (didn't look at it on my 1440p monitor at first, but noticed it when I did). I can try some kind of border :thinking: 

Looks like this for me rn: https://i.imgur.com/9zPvr4C.png

Reply:
doesn't seem to work probperly - i can't get it to produce what you see

(sorry for the missed commit - i commited on the wrong branch)

Reply:
I've just pushed an update that's fixing the button layout.

If this version works for you, too this will be ready to go.

![image](https://github.com/freqtrade/frequi/assets/5024695/7dbc54eb-3e93-438f-8804-d053ede8146f)

Reply:
Ah sorry, I was gonna fix it, but got busy with something else :laughing: Looks good to me though!